options/locale/locale_en-US.ini

```
@@ -659,7 +659,7 @@ comment_type_group_project = Project
 comment_type_group_issue_ref = Issue reference
 saved_successfully = Your settings were saved successfully.
 privacy = Privacy
-keep_activity_private = Hide the activity from the profile page
+keep_activity_private = Hide Activity from Profile
```

```suggestion
keep_activity_private = Hide Activity from Profile page
```

I think this is more clear, as otherwise people may need to guess what "Profile" means in gitea context.

---

Yes, but perhaps we should either keep the title case, or not use title case at all here.
I'm probably rather for dropping the title case, it's not a heading.

---

Agree, updated suggestion to all-lowercase.